test/intl402/Temporal/TimeZone/etc-timezone.js

```diff
@@ -10,27 +10,34 @@ features: [Temporal]
 // "Etc/GMT-0" through "Etc/GMT-14" are OK
 
 assert.sameValue(
-  Temporal.TimeZone.from("Etc/GMT-0").toString(),
+  (new Temporal.TimeZone("Etc/GMT-0")).toString(),
   "UTC", // if the offset is -0, we say "UTC" rather than "GMT"
   "Etc/GMT-0 is a valid timezone"
 );
 
 [1,2,3,4,5,6,7,8,9,10,11,12,13,14].forEach((n) => {
   let tz = "Etc/GMT-" + n;
-  let instance = Temporal.TimeZone.from(tz);
+  let instance = new Temporal.TimeZone(tz);
   assert.sameValue(
     instance.toString(),
     tz,
     tz + " is a valid timezone"
   );
 });
 
+let gmtMinus15TZ = "Etc/GMT-15";
+assert.throws(
+  RangeError,
+  () => { new Temporal.TimeZone(gmtMinus15TZ); },
+  gmtMinus15TZ + " is an invalid timezone"
+);
```

Comment:
This test will fail if IANA ever add Etc/GMT-15.
```suggestion
let gmtMinus24TZ = "Etc/GMT-24";
assert.throws(
  RangeError,
  () => { new Temporal.TimeZone(gmtMinus24TZ); },
  gmtMinus24TZ + " is an invalid timezone"
);
```

Reply:
Another of my previous review suggestions :sweat_smile: It didn't occur to me that IANA could add it in the future. I think checking the "out of bounds" case by using 24 is a good idea.